ecs/vulnerability-detector/fields/custom/package.yml

```diff
@@ -0,0 +1,14 @@
+- name: package
+  title: Package
+  group: 2
+  short: Fields to describe the package relevant to an event.
+  description: >
+    The package fields describe information about a package that is
+    relevant to an event.
+  type: group
+  fields:
+    - name: size
+      type: unsigned_long
+      level: custom
+      description: >
+        Size of the package.
```

Comment:
Keep the ECS description:
> Package size in bytes.

Reply:
> These fields contain information about an installed software package. It contains general information about a package, such as name, version or size. It also contains installation details, such as time or location.

Reply:
Fixed in commit https://github.com/wazuh/wazuh-indexer/pull/662/commits/dbbe061acd82bdb0f9fe4cf0d287dc8d2cc93e5c